index.mako

```diff
@@ -1124,6 +1124,40 @@ the commit will be aborted.
 [commit-msg]: https://git-scm.com/docs/githooks#_commit_msg
 [prepare-commit-msg]: https://git-scm.com/docs/githooks#_prepare_commit_msg
 
+
+_new in __VERSION__: pre-commit can be used to manage [post-checkout] hooks.
```

Comment:
this needs a header, probably something like `## pre-commit for switching branches`

the version will be 2.2.0 so you can amend that in right now